micronutrientsupport/micronutrient-support-tool
# Intervention table legends and intervention id selection

- only show legend information which matches the current page contents
- when selecting an intervention to load, dont allow the user to duplicate one already loaded (doesnt affect forking) closes #864 

## to test
- for each intervention page check that the legend is correct i.e. only show focus mn on pages with focus mn, no mg/kg units when not on the correct page etc
- add an intervention > load intervention, select one. Close dialog, then go back to add intervention and your currently selected interventions cannot be re-added
